src/components/Txt/index.tsx

```diff
@@ -107,6 +151,10 @@ export interface InternalTxtProps extends React.HTMLAttributes<HTMLElement> {
 	align?: Align;
 	/** If true, replace the text not contained in the container with three dots */
 	truncate?: boolean;
+	/** If true, it adds the copy icon and functionality at the end of a Txt component */
+	copy?: boolean;
+	/**  If true, the copy icon only appears when hovering over the content to be copied */
+	hoverCopy?: boolean;
```

Comment:
I would honestly keep it as `showCopyButton` maybe we can change it to `showCopyMode`, just in case we want to handle multiple states, and also to keep the same property we have in the Copy component.

```suggestion
	showCopyMode?: 'always' | 'hover';
```